redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+// ACTIONS
+const CHECK_STATUS = 'bookstore/src/redux/category/CHECK_STATUS';
```

Comment:
Kindly also stick here with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions